arch/arm/boot/dts/bcm2835-rpi-a-plus.dts

```diff
@@ -126,3 +126,8 @@
 	pinctrl-0 = <&uart0_gpio14>;
 	status = "okay";
 };
+
```

Comment:
[ I can't leave a comment against the commit message, so... ]

This commit modified the platform dts files. not the overlays.

Reply:
Ack. It was the mainline patch that was the one really still in progress when I did the push last thing Thursday.